lib/grape-swagger/endpoint.rb

```diff
@@ -192,14 +190,17 @@ def response_object(route, markdown)
 
         @definitions[response_model][:description] = description_object(route, markdown)
         # TODO: proof that the definition exist, if model isn't specified
-        memo[value[:code]][:schema] = if route.options[:is_array]
-                                        { 'type' => 'array', 'items' => { '$ref' => "#/definitions/#{response_model}" } }
-                                      else
-                                        { '$ref' => "#/definitions/#{response_model}" }
-                                      end
+        memo[value[:code]][:schema] = route.options[:is_array] ? { 'type' => 'array', 'items' => { '$ref' => "#/definitions/#{response_model}" } } : { '$ref' => "#/definitions/#{response_model}" }
```

Comment:
I couldn't see any other way to cut down the number of lines. A bit of a cheat I know. Happy to change it back, but Rubocop will complain, and I'm not sure there's any way to add the functionality without increasing the number of lines :/